cdap-docs/build.sh

```diff
@@ -460,18 +490,74 @@ function copy_docs_inner_level() {
 }
 
 function build_zip() {
+  local zip_target=${1}
+  local zip_extras=${2}
   set_project_path
-  make_zip ${1}
-}
-
-function zip_extras() {
-  if [[ "x${1}" == "x" ]]; then
-    return
+  set_version
+  if [ "x${1}" == "x" ]; then
+    ZIP_DIR_NAME="${PROJECT}-docs-${PROJECT_VERSION}"
+  else
+    ZIP_DIR_NAME="${PROJECT}-docs-${PROJECT_VERSION}-$1"
   fi
-  echo "Adding .htaccess file (404 file)"
-  rewrite ${SCRIPT_PATH}/${COMMON_SOURCE}/${HTACCESS} ${TARGET_PATH}/${PROJECT_VERSION}/.${HTACCESS} "<version>" "${PROJECT_VERSION}"
   cd ${TARGET_PATH}
-  zip -qr ${ZIP_DIR_NAME}.zip ${PROJECT_VERSION}/.${HTACCESS}
+  doc_change_py="${TARGET_PATH}/../tools/doc-change.py"
+
+  echo "Removing old directories and zips"
+  rm -rf ${TARGET_PATH}/base 
+  rm -rf ${TARGET_PATH}/current
+  rm -rf ${TARGET_PATH}/${PROJECT_VERSION}
+  rm -rf ${TARGET_PATH}/*.zip
+
+  echo "Creating base"
+  mkdir base
+  cp -r ${HTML} base/en
+
+  # Add a redirect index.html file
+  echo "${REDIRECT_EN_HTML}" > base/index.html
```

Comment:
Any reason this isn't just a file copy with the HTML checked in as a regular file, rather than the contents being in a variable? Is the variable sourced from a file and I'm just not seeing it?